src/libtego_ui/shims/ConversationModel.cpp

```diff
@@ -268,6 +273,152 @@ namespace shims
         }
     }
 
+    void ConversationModel::deserializeTextMessageEventToFile(const EventData &event, std::ofstream &ofile) const
+    {
+        auto &md = this->messages[this->messages.size() - event.messageData.reverseIndex];
+        switch (md.status)
+        {
+            case Received:
+                fmt::print(ofile, "[{}] <{}>: {}\n",
+                                    md.time.toString().toStdString(),
+                                    this->contact()->getNickname().toStdString(),
+                                    md.text.toStdString()); break;
+            case Delivered:
+                fmt::print(ofile, "[{}] <{}>: {}\n",
+                                    md.time.toString().toStdString(),
+                                    tr("me").toStdString(),
+                                    md.text.toStdString()); break;
```

Comment:
should this be refactored to be like in [line 308](https://github.com/blueprint-freespeech/ricochet-refresh/pull/92/files#diff-7fd014b73af577d0fc794178c304985f65024bb07745fb9376d6e9f4a6f272b5R308)